.ruby-version

```diff
@@ -1 +1 @@
-2.1.3
+2.3.1
```

Comment:
I see. Well the problem is that 2.1.3 does not compile with rvm on Debian (and presumably on the latest Ubuntu too), the other developers that don't use the VM need to install a different version and remember to switch to it every time, which is error prone (e.g. if I run bundle install from the wrong ruby version). I'm not familiar with vagrant, is it hard to update the ruby version there?